Linux/Build/Deployment/azure-pipelines.yml

```diff
@@ -0,0 +1,101 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - "Linux/Build/Deployment"
+    exclude:
+      - "*"
+
+pr:
+  paths:
+    include:
+      - "Linux/Build/Deployment"
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/DASD-9551_Verify_K8_Manifest_Apply
+    endpoint: SFA
+  pipelines:
+  - pipeline: build
+    source: das-azure-pipelines-agents-linux-build (Image Build)
+    branch: DASD-9551_Test_Agents
+    trigger:
+      branches:
+        - master
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+- group: RELEASE das-azure-pipelines-agent
+- group: Release Management Resources
+- name: ContainerImageName
+  value: azure-pipelines-build-agent-ubuntu-20
+- name: ImageTag
+  value: $(resources.pipeline.build.runName)
+
+stages:
+  - stage: DeployDev
+    variables:
+    - group: DEV Linux Build Agent
+    - group: Dev Management Resources
+    jobs:
+    - job: DeployManifest
+      displayName: DeployManifest
+      steps:
+        - template: /pipeline-templates/step/deploy.yml
+          parameters:
+            VerifyDeployment: true
+            Directory: $(System.DefaultWorkingDirectory)/Linux/Build/Deployment
+            AzureServiceConnection: SFA-DAS-DevTest-ARM
+        - pwsh: |
+            Start-Sleep -Seconds 120 # Do Better. This gives time for the old pods to remove themselves from the pool before AcceptanceTests.
+          displayName: Sleep
+  - stage: AcceptanceTests
+    dependsOn: DeployDev
+    jobs:
+    - job: DotNetAcceptanceTests
+      pool:
+        name: DAS - Continuous Integration Beta
+        demands:
+          - Agent.OS -equals Linux
+      displayName: RunTests
+      strategy:
+        matrix:
+          DotNet3.1:
+            FrameworkVersion: netcoreapp3.1
+          DotNet6:
+            FrameworkVersion: net6.0
+      steps:
+        - task: DotNetCoreCLI@2
```

Comment:
These steps should be split out into a step template since they're repeated for the Windows Build Agent.  Probably worth doing the same for Windows Deploy Agent to support eventual creation of Linux Deploy Agent.

Probably some value in adding a short comment to explain the test, following the Pester pattern of Describe > Context > It would keep it familiar :)

Reply:
👌 I've made that a step comment and added comments. 

I started with acceptance tests for the Windows Deploy Agent but there isn't the equivalent for what I've done on the build agents.
Having the deployment verification on the kubectl apply is the first layer of test compared to the fire and forget deployment before. I'm hoping we can identify some areas of headache and see if we can write AcceptanceTests for those in future tickets.